example/app/controllers/Login.scala

```diff
@@ -31,11 +32,14 @@ class Login @Inject() (override val wsClient: WSClient, override val conf: Confi
    * (see `Application.scala`).
    */
   def oauth2Callback = Action.async { implicit request =>
-    // processOauth2Callback()  // without Google group membership checks
-    processOauth2Callback(requiredGoogleGroups, groupChecker)  // with optional Google group checks
+     processOauth2Callback()  // without Google group membership checks
```

Comment:
Even a ropey example is better than none, I'll be happy to help improve the ropiness if we can leave something to that effect in here. Here at the Guardian at least, group checking should be the default so it definitely has a place in the example.